saga-demo/booking/booking/src/main/resources/application.yaml

```diff
@@ -19,4 +19,10 @@ spring:
     name: pack-booking
 alpha:
   cluster:
-    address: alpha-server.servicecomb.io:8080
+    address: localhost:8080
```

Comment:
As we use the domain name in the docker, so please do not change the setting here.
You may need to create a integration test for it.